src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/manufacturers.yml

```diff
@@ -0,0 +1,129 @@
+admin_manufacturers_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: 'PrestaShopBundle:Admin/Sell/Catalog/Manufacturer:index'
+    _legacy_controller: AdminManufacturers
+
+#
+# Manufacturer routes
+#
+admin_manufacturers_search:
+  path: /
+  methods: POST
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Sell\Catalog\Manufacturer:search'
+    _legacy_controller: AdminManufacturers
```

Comment:
You need to add some _legacy_link, see [documentation](https://devdocs.prestashop.com/1.7/development/architecture/migration-guide/controller-routing/) and https://github.com/PrestaShop/PrestaShop/pull/12787 example

Reply:
Ill keep that in mind, I guess it should be done only after whole page is migrated? (as for now i still have some work with addresses create/edit, double-list filtering and so on...)

Reply:
If you guarantee this will not be forgotten, it's OK for me 😉